packages/jest-leak-detector/src/index.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+'use strict';
+
+const v8 = require('v8');
+const vm = require('vm');
+const weak = require('weak');
+
+// Setting the flag has no effect on the main context, but will allow new
+// contexts to access the garbage collector.
+v8.setFlagsFromString('--expose-gc');
```

Comment:
I'm not a fan of doing this here as a side-effect when the file is loaded. This means that we may expose the garbage collector simply by loading this module. We do use inline requires so it shouldn't happen but I'm sure at some point it does and it loads stuff it shouldn't. Can we move this to the first use of this module or so?

Reply:
Yes, makes sense.